src/block-types/unsupported-block/edit.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @format
+ * @flow
+ */
+
+import React from 'react';
+import { View, Text, TouchableWithoutFeedback } from 'react-native';
+import RCTAztecView from 'react-native-aztec';
+import type { BlockType } from '../../store/';
+
+type PropsType = BlockType & {
+	onChange: ( clientId: string, attributes: mixed ) => void,
+	onToolbarButtonPressed: ( button: number, clientId: string ) => void,
+	onBlockHolderPressed: ( clientId: string ) => void,
+};
+
+type StateType = {
+	selected: boolean,
+	focused: boolean,
+};
+
+// Styles
+import styles from '../../block-management/block-holder.scss';
+
+// Gutenberg imports
+import { Component } from '@wordpress/element';
+import { getBlockType } from '@wordpress/blocks';
+import { getUnknownTypeHandlerName } from '@wordpress/blocks';
+
+export default class UnsupportedBlockEdit extends React.Component<PropsType, StateType> {
+	constructor( props: PropsType ) {
+		super( props );
+	}
+
+	render() {
+		let blockName = this.props.name.charAt(0).toUpperCase() + this.props.name.slice(1);
```

Comment:
Wouldn’t this always be “Gmobile/unsupported” now?

Reply:
Yeppers.  I can just remove it if it feels redundant, although I left it in cos otherwise the block feels a bit empty at this time.

Let me know which way you prefer it.

Reply:
Since the other line already says unsupported, I think we can remove it for now